src/js/model.ts

```diff
@@ -93,9 +85,7 @@ export class Model extends (InMemoryEntity as Base) implements BaseModel {
 
     get Method(): Method {
         if (!this._method) {
-            const methodOrConfig = this.method;
-            const config = methodOrConfig || this.defaultMethodConfig;
-            this._method = this._MethodFactory.create(config);
+            this._method = this._MethodFactory.create(this.defaultMethodConfig);
```

Comment:
Should be `this._MethodFactory.create(this.method)`

and this.method should be set in constructor if not passed